setup.cfg

```diff
@@ -0,0 +1,68 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+[metadata]
+name = cylc
+author = Hilary Oliver
+author_email = cylc@googlegroups.com
+url=https://cylc.github.io/
+description =
+    Cylc (\"silk\") is a workflow engine for cycling systems -
+    it orchestrates distributed suites of interdependent
+    cycling tasks that may continue to run indefinitely.
+keywords =
+    cycling-workflows
+    hpc
+    job-scheduler
+    metascheduler
+    workflow-automation
+    workflow-engine
+    workflow-management
+    scheduling
+license = GPL
+license_file = COPYING
+platforms = any
+classifiers =
+    Environment :: Console
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Atmospheric Science
+python_requires = >=3.7
+setup_requires=pytest-runner==4.4
+
+[bdist_wheel]
+universal = 1
+
+[bdist_rpm]
+requires =
+    python3-colorama
+    python-isodatetime
```

Comment:
I guess this is something we need to address later?

Reply:
(Should be *python3-isodatetime*.)

Reply:
>I guess this is something we need to address later?

Oh, well thought. Yup, or we could change that right now, but it [hasn't changed in `isodatetime` yet I think](https://github.com/metomi/isodatetime/blob/b6a0eaf091c68ba2f4826fefbe69bbc35f52a747/setup.py#L60). So I think later we can change that here once we change it there.